src/components/ClientDetails/ClientProfileTable.jsx

```diff
@@ -0,0 +1,82 @@
+    
+    return (
+        <Wrapper>
+            <WrapperAccountCircleIcon> 
+                {clientProfilePhoto ? <ProfileImage imageSrc={img}/> : <AccountCircleIcon />}
+            </WrapperAccountCircleIcon>
+            <WrapperClientProfileDetails> 
+                <WrapperClientProfileDetailsItem>
```

Comment:
We should be creating components using styled components for styling rather than adding class names within a styled component. 

So instead of:
```
<WrapperClientProfileDetailsItem>
    <p className="client-details-field-title">First name:</p>
    <p className="client-details-field-data">{"Ashley"}</p>
 </WrapperClientProfileDetailsItem>
```

It should just be:

```
const FieldTitle = styled.p`
  font-weight: bold;
  font-size: 14px;
  display: inline;
`
const FieldData = styled.p`
  font-size: 14px;
  display: inline;
`

<WrapperClientProfileDetailsItem>
    <FieldTitle>First name:</FieldTitle>
    <FieldData>{"Ashley"}</FieldData>
 </WrapperClientProfileDetailsItem>
```
 
It just keeps our styling consistent

Reply:
DONE